BondInventoryControl/src/control/Menu.java

```diff
@@ -0,0 +1,25 @@
+package control;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+public class Menu {
```

Comment:
javadocコメントをつけてください。
（ここに限らず、すべてのソースコードで）
```suggestion
/**
   * ~~するクラス
   */
public class Menu {
```